addon/-private/system/model/internal-model.js

```diff
@@ -849,4 +851,21 @@ InternalModel.prototype = {
 
     return reference;
   }
-};
+}
+
+if (isEnabled('ds-reset-attribute')) {
+  /*
+     Returns the latest truth for an attribute - the canonical value, or the
+     in-flight value.
+
+     @method latestTrueValue
+     @private
+  */
+  InternalModel.prototype.latestTrueValue = function latestTrueValue(key) {
```

Comment:
I could settle for `lastCanonicalOrInFlightAttributeValue`. What about `latestSavedValue` or `latestComittedValue`? Since it's not necessarily canonical yet, but it's trying to be.